Sources/SWBTestSupport/CoreTestSupport.swift

```diff
@@ -50,7 +50,13 @@ extension Core {
         // When this code is being loaded directly via unit tests, find the running Xcode path.
         //
         // This is a "well known" launch parameter set in Xcode's schemes.
-        let developerPath = getEnvironmentVariable("XCODE_DEVELOPER_DIR_PATH").map(Path.init)
+        let developerPath: Path?
+        if let xcodeDeveloperDirPath = getEnvironmentVariable("XCODE_DEVELOPER_DIR_PATH").map(Path.init) {
+            developerPath = xcodeDeveloperDirPath
+        } else {
+            // In the context of auto-generated package schemes, try to infer the active Xcode.
+            developerPath = getEnvironmentVariable("PATH")?.components(separatedBy: ":").first.map(Path.init)?.dirname.dirname
```

Comment:
You could maybe try validating that the path contains ".app/Contents/Developer" before returning it?

Reply:
Pushed a change where I am taking a peek at version.plist